src/CMakeLists.txt

```diff
@@ -87,6 +89,21 @@ set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} ${c_cxx_flags}")
 if(BUILD_FAST_TOKENIZERS)
   # FastTokenizers use static MSVC runtime, hence we have to use it as well
   set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
+
+  if(WIN32 AND CMAKE_BUILD_TYPE MATCHES Debug)
+    set(CMAKE_DEBUG_POSTFIX "d")
```

Comment:
I suppose debug postfix should be set unconditionally of current cmake build type

See https://github.com/openvinotoolkit/openvino/blob/master/cmake/developer_package/OpenVINODeveloperScriptsConfig.cmake#L148-L168